src/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -114,6 +114,9 @@
     <Compile Include="$(CommonPath)\System\Net\HttpDateParser.cs">
       <Link>Common\System\Net\HttpDateParser.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\System\Runtime\CompilerServices\PreserveDependencyAttribute.cs">
```

Comment:
I do not see any use of this. Where is it needed?

Reply:
I'll delete it.  It's a holdover from before I added the ToString overrides to each of the data classes, at which point nothing was using the property getters and I needed this to keep them alive.  Now ToString takes care of that.